HongAndJerry/HongAndJerry/Source/Presentation/Export/AlbumManager.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  AlbumManager.swift
+//  HongAndJerry
+//
+//  Created by Hong on 7/18/25.
+//
+
+import Photos
+
+final class AlbumManager: AlbumRepository {
+    
+    func checkAlbum(named title: String) throws -> PHAssetCollection {
+        if let album = fetchAlbum(title: title) { return album }
```

Comment:
P2
`if let existingAlbum`으로 바꾸면 이미 있는 엘범을 체크하는 로직인걸 확인하기 쉬울 것 같아요
```suggestion
        if let existingAlbum = fetchExistingAlbum(title: title) { return album }
```